oauthproxy_test.go

```diff
@@ -1342,11 +1344,6 @@ func TestUserInfoEndpointAccepted(t *testing.T) {
 		Email: "john.doe@example.com", AccessToken: "my_access_token"}
 	err = test.SaveSession(startSession)
 	assert.NoError(t, err)
-
-	test.proxy.ServeHTTP(test.rw, test.req)
-	assert.Equal(t, http.StatusOK, test.rw.Code)
-	bodyBytes, _ := ioutil.ReadAll(test.rw.Body)
-	assert.Equal(t, "{\"email\":\"john.doe@example.com\"}\n", string(bodyBytes))
```

Comment:
Good catch -- no idea what happened there.

Definitely seems like a mistake since I extended this test in this PR: https://github.com/oauth2-proxy/oauth2-proxy/pull/850/files#diff-5391075a81db21342eb96dd272ba55de4a981f521b6bc8647d1ab8be5d5d5754R1281